src/power_grid_model/core/power_grid_model.py

```diff
@@ -283,54 +289,63 @@ def calculate_power_flow(
+                calculation method is iterative.
+            max_iterations (int, optional): Maximum number of iterations, applicable only when the calculation method
+                is iterative.
+            calculation_method (an enumeration or string): The calculation method to use.
+
+                - Newton_raphson: Use Newton-Raphson iterative method (default).
+                - Linear: Use linear method.
+            update_data (dict, optional):
+                None: Calculate power flow once with the current model attributes.
+                Or a dictionary for batch calculation with batch update.
+
+                    - key: Component type name to be updated in batch.
+                    - value:
+
+                        - For homogeneous update batch (a 2D numpy structured array):
+
+                            - Dimension 0: Each batch.
+                            - Dimension 1: Each updated element per batch for this component type.
+                        - For inhomogeneous update batch (a dictionary containing two keys):
+
+                            - indptr: A 1D integer numpy array with length n_batch + 1. Given batch number k, the
+                              update array for this batch is data[indptr[k]:indptr[k + 1]]. This is the concept of
+                              compressed sparse structure.
+                              https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.csr_matrix.html
+                            - data: 1D numpy structured array in flat.
+            threading (int, optional): Applicable only for batch calculation.
+
+                - < 0: Sequential
+                - = 0: Parallel, use number of hardware threads
+                - > 0: Specify number of parallel threads
+            output_component_types ({set, list}, optional): List or set of component types you want to be present in
+                the output dict. By default, all component types will be in the output.
+            continue_on_batch_error (bool, optional): If the program continues (instead of throwing error) if some
+                scenarios fail.
 
         Returns:
-            dictionary of results of all components
-                key: component type name to be updated in batch
-                value:
-                    for single calculation: 1D numpy structured array for the results of this component type
-                    for batch calculation: 2D numpy structured array for the results of this component type
-                        Dimension 0: each batch
-                        Dimension 1: the result of each element for this component type
-            Error handling:
-                in case an error in the core occurs, an exception will be thrown
+            Dictionary of results of all components.
+
+                - Key: Component type name to be updated in batch.
+                - value:
```

Comment:
inconsistent capitalization.

looking at the other `key`, `value` combinations, the `key` should be lower-case

```suggestion
                - key: Component type name to be updated in batch.
                - value:
```